index.html

```diff
@@ -61,8 +61,6 @@ <h2 class="prof">I’m a Software Developer</h2>
     <h3 class="works-h3">My Recent Work</h3>
     <hr class="works-hr">
   </div>
-  
-
 
 <div class="box-container">
```

Comment:
- [ ] As suggested by the previous reviwer, the main `index.html` content should be dynamically generated using javascript based on project requirements. I can see headings and skills section still static here.
- You can use [This link](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and[This](https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636) to see sample implementaion

Reply:
@Shedrack-Sunday 
i have created the whole modal and grids dynamically using Javascript, can you please show me the modal section in my html?